ios/FluentUI.Demo/FluentUI.Demo/Demos/ShimmerLinesViewDemoController.swift

```diff
@@ -60,27 +60,29 @@ class ShimmerViewDemoController: DemoController {
         }
 
         container.addArrangedSubview(shimmerViewLabel("A ShimmerLinesView needs no containerview or subviews"))
-        container.addArrangedSubview(Separator())
+        container.addArrangedSubview(dividers[0].view)
         container.addArrangedSubview(ShimmerLinesView())
-        container.addArrangedSubview(Separator())
+        container.addArrangedSubview(dividers[1].view)
 
         container.addArrangedSubview(shimmerViewLabel("ShimmerView shimmers all the top level subviews of its container view"))
-        container.addArrangedSubview(Separator())
+        container.addArrangedSubview(dividers[2].view)
         container.addArrangedSubview(shimmeringContentView(false))
-        container.addArrangedSubview(Separator())
+        container.addArrangedSubview(dividers[3].view)
 
         container.addArrangedSubview(shimmerViewLabel("With shimmersLeafViews set, the ShimmerView will shimmer the labels inside the stackview"))
-        container.addArrangedSubview(Separator())
+        container.addArrangedSubview(dividers[4].view)
         container.addArrangedSubview(shimmeringContentView(true))
-        container.addArrangedSubview(Separator())
+        container.addArrangedSubview(dividers[5].view)
 
         container.addArrangedSubview(shimmerViewLabel("Revealing style shimmer on an image: the gradient reveals its container view as it moves"))
-        container.addArrangedSubview(Separator())
+        container.addArrangedSubview(dividers[6].view)
         container.addArrangedSubview(shimmeringImageView(.revealing))
 
-        container.addArrangedSubview(Separator())
+        container.addArrangedSubview(dividers[7].view)
         container.addArrangedSubview(shimmerViewLabel("Concealing style shimmer on an image: the gradient conceals its container view as it moves"))
-        container.addArrangedSubview(Separator())
+        container.addArrangedSubview(dividers[8].view)
         container.addArrangedSubview(shimmeringImageView(.concealing))
     }
+
+    private let dividers = [MSFDivider(), MSFDivider(), MSFDivider(), MSFDivider(), MSFDivider(), MSFDivider(), MSFDivider(), MSFDivider(), MSFDivider()]
```

Comment:
I'm not a huuuuge fan of this line... any way we can simplify it with a loop and a `.map` or something similar?

Reply:
I tried using this, https://developer.apple.com/documentation/swift/array/1641692-init, but that just made an array with nine copies of the same divider, so not quite was I was looking for. Wasn't sure if there was another way, but I'll look in to .map.

Reply:
There might be examples I wrote in the macOS button example page. I can't remember if we refactored them out